test/Dialect/FIRRTL/infer-resets.mlir

```diff
@@ -1069,3 +1069,30 @@ firrtl.circuit "InferToRWProbe" {
    firrtl.connect %r_b, %driver : !firrtl.reset, !firrtl.asyncreset
   }
 }
+
+// -----
+
+// CHECK-LABEL: "RWProbeOp"
+firrtl.circuit "RWProbeOp" {
+  // CHECK: <a: asyncreset, b: asyncreset>
+  // CHECK-NOT: firrtl.reset
+  firrtl.module @RWProbeOp(in %driver: !firrtl.asyncreset, out %out: !firrtl.bundle<a: reset, b: reset>, out %out2: !firrtl.reset) {
+    %r = firrtl.wire sym [<@r,0,public>,<@r_a,1,public>,<@r_b,2,public>] : !firrtl.bundle<a: reset, b flip: reset>
```

Comment:
Nit, no `CHECK` for other `!firrtl.asyncreset`s  here ?

Reply:
Was mostly leaving that to verifiers, but checking explicitly sounds good.  Updated!